.github/workflows/_create-release-pr.yml

```diff
@@ -53,9 +53,10 @@ jobs:
             || inputs.component-name == 'Compute' && 'release-compute'
           }}
       run: |
-        today=$(date +'%Y-%m-%d')
+        today=$(date -u +'%Y-%m-%d')
+        now=$(date -u +'%Y-%m-%d_%H-%M_UTC')
         echo "title=${COMPONENT_NAME} release ${today}" | tee -a ${GITHUB_OUTPUT}
```

Comment:
should we have ${now} here as well? or could be both: ${today} for readability and ${now} to distinguish releases

Reply:
> could be both: ${today} for readability and ${now} to distinguish releases

Yeah, that was exactly my thoughts. Time doesn't look nice in PR title :(
I've changed `now` from datetime to time, so now there's slightly less repetition